motion/p5js/motion-without-motion/index.js

```diff
@@ -0,0 +1,55 @@
+  function setup() {
+      createCanvas(400,400);
```

Comment:
The canvas should be 500 by 500 and assigned to the sketch id div. See https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-P5js-Sketches#code